### PR TITLE
volume size

### PR DESCRIPTION
**Pull request by harden71**

### CODE REVIEW

File: ec2-backup.py [hunk 1]
```diff
@@ -7,6 +7,13 @@
 import boto.ec2
 import optparse
 
+#calculate the volume size
+def du(path):
+    return subprocess.check_output(['du','-sh', path]).split()[0].decode('utf-8')
+
+if __name__ == "__main__":
```
Comment (phunold) [inline ec2-backup.py]: I may slightly change up the du.
we need to get the size in GB.

**Comment by phunold** [comment]
I may slightly change up the du.
we need to get the size in GB.

**Comment by harden71** [comment]
Ok you know how to do it or you want me to do it?

**Comment by phunold** [comment]
send me a revised version where it puts out size in GB, and I will just add 1 to it. Amazon has GB as lowest unit.
thank you.

**Comment by harden71** [comment]
Tomorrow morning philip or maybe later today if i get home early

**Pull request merged**